ubuntu/gpu/README.md

```diff
@@ -1,37 +1,33 @@
 # GPU Containers
 
-There are two variations of GPU containers that can be used depending upon the CUDA version you wish to use:
-[`cuda-10.1`](cuda-10.1) contains the layers which install CUDA 10.1
-[`cuda-11.0`](cuda-11.0) contains the layers which install CUDA 11.0
+**WARNING**: Using conda in DCS images is no longer supported starting Databricks Runtime 9.0. We highly recommend users to extend [`cuda-11.8`](cuda-11.8) examples.
+ We no longer support [`cuda-10.1`](cuda-10.1) and [`cuda-11.0`](cuda-11.0) compatibility with latest databricks runtime.
 
+**WARNING**: DCS images which extends [`venv`](cuda-11.8/venv/) need to be paired with DBR 14.x when creating a
```

Comment:
```suggestion
```suggestion
**WARNING**: DCS images which extends [`venv`](cuda-11.8/venv/) need to be paired with DBR 14.x when creating a
```
**WARNING**: DCS images which extends [`venv`](cuda-11.8/venv/) need to be paired with DBR 14.0 and above when creating a
```